src/components/CanvasLayers.js

```diff
@@ -30,11 +30,11 @@ const reorder = (list, startIndex, endIndex) => {
 /** */
 export class CanvasLayers extends Component {
   /** */
-  static getUseableLabel(resource, index) {
+  static getUseableLabel(resource, index, langs) {
     return (resource
       && resource.getLabel
       && resource.getLabel().length > 0)
-      ? resource.getLabel().getValue()
+      ? resource.getLabel().getValue(langs)
```

Comment:
Can you remind me of the manifesto behavior if there's no label in a given language? Is there a fallback or something?

Reply:
Currently the behavior is to simply pick the first value, with no regard for the (legacy) `defaultLocale`. I'm still planning on submitting a PR that uses that instead, but haven't gotten around to it yet.